.claude/skills/code-review/SKILL.md

```
@@ -47,25 +55,24 @@ Notes:
 
 ---
 
 ### Code review
 
-Found 3 issues:
+#### High confidence (score ≥ 80)
 
-1. <brief description of bug> (CLAUDE.md says "<...>")
+| # | Finding | Score | Disposition |
+|---|---------|-------|-------------|
+| 1 | <brief description> (CLAUDE.md says "<...>") — [link](https://github.com/anthropics/claude-code/blob/1d54823877c4de72b2316a64032a54afc404e619/README.md#L13-L17) | 85 | **Fix** — must address before merge |
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)